docs/source/error-codes/self-service/index.rst

```diff
@@ -11,81 +11,186 @@ Self-Service Error Codes (Experimental)
 
 
 Self-Service Error Codes Migration Guide
-########################################
+----------------------------------------
 
-The introduction of the self-service error codes means that some of the gRPC error codes returned from service methods change.
-Consult the table below for details on those changes.
+The introduction of the self-service error codes means that some of the gRPC error codes returned from service endpoints change.
+Consult the sections below for details on those changes.
 
-Note that the table below contains entries only for cases where a gRPC codes was changed.
+Note that the tables below contain entries only for cases where a gRPC codes was changed.
 For example, a service method previously returning gRPC error code ``CODE_A`` may now return 
```

Comment:
For consistency, we should either use `service method` or `service endpoint` throughout this document.